force-app/tests/Testing Recipes/PSGTestingRecipes.cls

```diff
@@ -2,11 +2,11 @@
 private class PSGTestingRecipes {
     @isTest
     static void testRecalculatePSG() {
-        // get the PSG by name (may have been modified in deployment)
+        // get the PSG by name
         PermissionSetGroup psg = [
             SELECT Id, Status
             FROM PermissionSetGroup
-            WHERE DeveloperName = 'SalesPersona'
+            WHERE DeveloperName = 'Permission_Set_Group_for_testing'
```

Comment:
@pozil I had to modify this code to fix a test failure.

Looks like winter 24 orgs removed a out-of-box permission set group `SalesPersona` this class was assuming. I created a new one so we do not depend on anything existing.